<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>851</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>851</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="eLastname">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>180</width>
      <height>23</height>
     </rect>
    </property>
    <property name="maxLength">
     <number>40</number>
    </property>
    <property name="placeholderText">
     <string comment="Введите фамилию">Фамилия</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="eName">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>10</y>
      <width>180</width>
      <height>23</height>
     </rect>
    </property>
    <property name="maxLength">
     <number>40</number>
    </property>
    <property name="placeholderText">
     <string>Имя</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="eSurname">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>10</y>
      <width>180</width>
      <height>23</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Отчество</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="eAge">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>113</width>
      <height>23</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>113</width>
      <height>23</height>
     </size>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhDigitsOnly</set>
    </property>
    <property name="maxLength">
     <number>2</number>
    </property>
    <property name="placeholderText">
     <string>Возраст</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cbGender">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>40</y>
      <width>121</width>
      <height>24</height>
     </rect>
    </property>
    <property name="currentText">
     <string/>
    </property>
    <property name="placeholderText">
     <string>Выберите пол</string>
    </property>
    <item>
     <property name="text">
      <string>Мужской</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Женский</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="cbTest">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>681</width>
      <height>24</height>
     </rect>
    </property>
    <property name="maxCount">
     <number>30</number>
    </property>
    <property name="placeholderText">
     <string>Выберите тест</string>
    </property>
    <item>
     <property name="text">
      <string>1. Опросник К.Леонгарда - Г.Шмишека для исследования акцентированных свойств личности</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2. Шкала личностной тревожности для учащихся 10-16 лет</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>3. ТЕСТ КЕТТЕЛЛА, ДЕТСКИЙ ВАРИАНТ. АДАПТИРОВАН Э. М. АЛЕКСАНДРОВСКОЙ. 12 ФЛО-120</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>4. ШКАЛА СОЦИАЛЬНОЙ ПОДДЕРЖКИ (МНОГОМЕРНАЯ ШКАЛА ВОСПРИЯТИЯ СОЦИАЛЬНОЙ ПОДДЕРЖКИ – MSPSS ; Д.ЗИМЕТ; АДАПТАЦИЯ В. М. ЯЛТОНСКИЙ, Н. А. СИРОТА)</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="pbBeginTest">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>70</y>
      <width>141</width>
      <height>24</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>141</width>
      <height>0</height>
     </size>
    </property>
    <property name="text">
     <string>Начать тестирование</string>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>831</width>
      <height>441</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QProgressBar" name="progressBar">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>413</y>
       <width>811</width>
       <height>20</height>
      </rect>
     </property>
     <property name="value">
      <number>0</number>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="pteInstruction">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>811</width>
       <height>101</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::StrongFocus</enum>
     </property>
     <property name="inputMethodHints">
      <set>Qt::ImhNone</set>
     </property>
     <property name="undoRedoEnabled">
      <bool>true</bool>
     </property>
     <property name="lineWrapMode">
      <enum>QPlainTextEdit::WidgetWidth</enum>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="plainText">
      <string/>
     </property>
     <property name="backgroundVisible">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QWidget" name="widget" native="true">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>811</width>
       <height>391</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>851</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
